tests/lock_test.py

```diff
@@ -0,0 +1,129 @@
+"""Tests for the lock client"""
+
+import pytest
+import asyncio
+from client import client
+
+@pytest.mark.asyncio
+async def test_lock_unlock_should_success_in_normal_path():
+    """
+    Test lock unlock should success in normal path
+    """
+    curp_members = ["172.20.0.3:2379", "172.20.0.4:2379", "172.20.0.5:2379"]
+    cli = await client.Client.connect(curp_members)
+    lock_client = cli.lock_client
```

Comment:
Maybe we can use a test fixture to make this test case more brief. FYI: https://stackoverflow.com/questions/49936724/async-fixtures-with-pytest

Reply:
We will try to use this in after pr.